[#5] Messages/Alerts

LavanyaThuyamani opened the pull request:
Implements styles for messages / alerts with the help of figma design style guide.

LavanyaThuyamani committed:
[#5] Messages/Alerts

Implements styles for messages / alerts with the help of figma design style guide.

mageshravi commented on css/src/_m-message.scss:
```diff
@@ -0,0 +1,72 @@
+    &__item {
+        --message-txt: #da394c;
+        --message-bg: rgba(238, 162, 171, 0.25);
+        position: relative;
```
Add empty line between css variables and properties.

LavanyaThuyamani replied:
Added an empty line between css variables and properties.

mageshravi commented on css/src/_m-message.scss:
```diff
@@ -0,0 +1,72 @@
+        margin-inline-end: 11px;
+        margin-block-end: 20px;
+        padding: 10px 15px;
+
```
Remove this empty line.

LavanyaThuyamani replied:
Removed the empty line in the 14th line.

LavanyaThuyamani committed:
Rename _m-message.scss to _m-messages.scss

mageshravi commented on css/src/_m-message.scss:
```diff
@@ -0,0 +1,72 @@
+    }
+}
+
+@media (prefers-color-scheme: dark) {
```
Move this block inside the existing `.m-messages` block like shown below,

```scss
.m-messages {
    /* ... */

    @media (prefers-color-scheme: dark) {
        &__item {
            /* ... */
        }
    }
}

LavanyaThuyamani replied:
Moved the @media block inside the existing `.m-messages` block like the above shown structure.

mageshravi commented on index.html:
```diff
@@ -74,6 +74,62 @@ <h6>Heading 6</h6>
+        <li class="m-messages__item">
+            <img class="m-messages__close-btn" src="close-btn.svg" alt="close" />
+
+            <span class="m-messages__desc">This is an error message
+                that flows two lines full.</span>
```
Remove line break in this line.

LavanyaThuyamani replied:
Removed the line break in this line.

mageshravi commented on index.html:
```diff
@@ -74,6 +74,62 @@ <h6>Heading 6</h6>
+        <li class="m-messages__item m-messages__item--warning">
+            <img class="m-messages__close-btn" src="close-btn.svg" alt="close" />
+
+            <span class="m-messages__desc">This is a warning
+                that flows two lines full.</span>
```
Remove line break.

LavanyaThuyamani replied:
Removed the line break in this line.

mageshravi commented on index.html:
```diff
@@ -74,6 +74,62 @@ <h6>Heading 6</h6>
+        <li class="m-messages__item m-messages__item--info">
+            <img class="m-messages__close-btn" src="close-btn.svg" alt="close" />
+
+            <span class="m-messages__desc">This is an information
+                that flows two lines full.</span>
```
Remove line break.

LavanyaThuyamani replied:
Removed the line break in this line.

mageshravi reviewed (changes requested):
Changes requested

LavanyaThuyamani committed:
[#5] Messages/Alerts

Modifies the changes that are required.

LavanyaThuyamani committed:
Merge branch 'feature/gh-5/Messages/Alerts' of github.com:Webinative/webinative-styleguide into feature/gh-5/Messages/Alerts

LavanyaThuyamani committed:
[#5] Messages/Alerts

Adds the close button image

LavanyaThuyamani committed:
[#5] Messages/Close-btn

Adds the close button to the css folder destination instead of using it as an image.

LavanyaThuyamani committed:
Delete close-btn.svg
